thirdeye-ui/src/app/components/alert-wizard-v2/alert-template/alert-template-form-field/form-component-for-template-field/form-component-for-template-field.component.tsx

```diff
@@ -134,7 +134,12 @@ export const FormComponentForTemplateField: FunctionComponent<FormComponentForTe
                     <Grid item>
                         <Switch
                             data-testid={`switch-${propertyKey}`}
-                            defaultChecked={!!value}
+                            defaultChecked={
+                                value === undefined
+                                    ? templateFieldProperty.defaultValue ===
```

Comment:
Not a blocker. Ideally, you would use a platform util to parse a string to boolean. The value could change from "true" to "True" and things will break.